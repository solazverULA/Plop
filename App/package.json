{
  "name": "NotificatorListener",
  "version": "0.0.1",
  "private": true,
  "scripts": {
    "start": "node node_modules/react-native/local-cli/cli.js start",
    "test": "jest",
    "jest-update-snapshot": "jest --updateSnapshot",
    "androidRun1": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res && react-native run-android --deviceId SSIFSCFIMRBU95JV",
    "androidRun2": "react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res && react-native run-android "
  },
  "dependencies": {
    "react": "16.0.0",
    "react-native": "0.70.6",
    "react-native-background-job": "^2.1.1",
    "react-native-elements": "^0.18.5",
    "react-native-linear-gradient": "^2.4.0",
    "react-native-modal": "^4.1.1",
    "react-native-phone-input": "^0.1.10",
    "react-native-push-notification": "^3.0.2",
    "react-native-vector-icons": "^4.5.0",
    "react-navigation": "^1.0.0-beta.23"
  },
  "devDependencies": {
    "babel-jest": "22.0.4",
    "babel-preset-react-native": "4.0.0",
    "jest": "22.0.4",
    "react-test-renderer": "16.0.0"
  },
  "jest": {
    "preset": "react-native"
  }
}
