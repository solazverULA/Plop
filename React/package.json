{
  "name": "nud-react",
  "version": "1.0.0",
  "private": true,
  "homepage": "https://timcreative.github.io/now-ui-dashboard-react",
  "dependencies": {
    "animate.css": "^3.6.1",
    "babel-preset-env": "^1.7.0",
    "babel-preset-react": "^6.24.1",
    "bootstrap": "4.0.0-beta",
    "chart.js": "^2.7.2",
    "history": "4.7.2",
    "jquery": "3.3.1",
    "node-sass": "^4.9.0",
    "node-sass-chokidar": "^1.3.0",
    "npm-run-all": "4.1.1",
    "perfect-scrollbar": "1.3.0",
    "react": "16.1.0",
    "react-animated-css": "^1.0.4",
    "react-chartjs-2": "2.7.0",
    "react-dom": "16.1.0",
    "react-google-maps": "9.2.2",
    "react-notification-alert": "0.0.7",
    "react-router-dom": "4.2.2",
    "react-scripts": "^1.1.4",
    "reactstrap": "5.0.0-alpha.4"
  },
  "devDependencies": {
    "babel-jest": "^22.0.4",
    "jest": "^22.0.4",
    "react-test-renderer": "^16.0.0"
  },
  "scripts": {
    "build-css": "node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/",
    "watch-css": "npm run build-css && node-sass-chokidar --include-path ./src --include-path ./node_modules src/ -o src/ --watch --recursive",
    "start-js": "react-scripts start",
    "start": "npm-run-all -p watch-css start-js",
    "build": "npm run build-css && react-scripts build",
    "test": "jest",
    "jest-update-snapshot": "jest --updateSnapshot",
    "eject": "react-scripts eject"
  }
}
