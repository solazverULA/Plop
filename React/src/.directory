[Dolphin]
Timestamp=2018,6,27,3,57,52
Version=3
