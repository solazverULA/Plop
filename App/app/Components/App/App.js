/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 * @flow
 */
import Login from "../Login/Login";
import Home from "../Home/Home";
import Api from "../../Api/Api";






import React, { Component } from 'react';
import {
  Platform,
  StyleSheet,
  Text,
  View,
  TextInput,
  Button,
  Alert,
  AsyncStorage
} from 'react-native';



export default class App extends Component<{}> {
  constructor(props) {
    super(props);
  
    this.state = {isloging:false};
   /*AsyncStorage.getItem('ListenerData')
      .then((data)=>{this.setState({isloging:data!=null})})
      .catch((errr)=>Alert.alert("nno", JSON.stringify(errr)))*/

      
   

  }
  render() {
    return (
      (this.state.isloging 
        ? <Home _logout={()=>{
            this.setState({isloging:false});
          
             }}/> 
        : <Login _goHome={()=>this.setState({isloging:true})}/>)
    );
  }
}


