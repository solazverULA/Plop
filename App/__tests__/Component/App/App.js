import 'react-native';
import React from 'react';
import App from '../../../app/Components/App/App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const loading = renderer.create(
    <App />
  ).toJSON();
  
  expect(loading).toMatchSnapshot();
});
