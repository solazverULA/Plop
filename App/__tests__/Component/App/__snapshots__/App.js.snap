// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "white",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "fontSize": 22,
        "fontWeight": "bold",
        "margin": 10,
        "textAlign": "center",
      }
    }
  />
  <View
    style={
      Array [
        Object {},
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {},
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          undefined,
          Array [
            Object {
              "color": "#86939e",
              "fontSize": 15,
              "fontWeight": "bold",
              "marginBottom": 1,
              "marginLeft": 20,
              "marginRight": 20,
              "marginTop": 15,
            },
            undefined,
            undefined,
          ],
        ]
      }
    >
       NÃºmero de telefono
    </Text>
  </View>
  <View>
    <View
      style={
        Array [
          Object {},
          undefined,
        ]
      }
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Array [
            Object {},
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            undefined,
            Array [
              Object {
                "color": "#ff190c",
                "fontSize": 15,
                "marginBottom": 1,
                "marginLeft": 20,
                "marginRight": 20,
                "marginTop": 5,
              },
              undefined,
              undefined,
            ],
          ]
        }
      >
        
      </Text>
    </View>
  </View>
</View>
`;
